lib/gooddata/core/splunk_logger.rb

```diff
@@ -0,0 +1,37 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+module GoodData
+  # Dummy implementation of logger
+  class SplunkLogger < Logger
+
+    def initialize(input = STDERR)
+      super(input)
+      @formatter = Logger::Formatter.new
+
+      @logs = ""
+    end
+
+    def clear_logs
+      @logs = ""
+    end
+
+    def log(log, time, buffered=true)
```

Comment:
Style/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.